meshapp/src/pages/Profiles/Page.tsx

```diff
@@ -28,6 +29,7 @@ import ProfileHeader from "./ProfileHeader";
  * @param {Education} props.education - an array containing objects that each contain a degree,school, and description
  * @param {Experience} props.experience - an array contains objects that each contain a occupation name, organization, and description
  * @param {number} props.accountID - ID of Profile account
+ * @param {boolean} props.viewOnly - whether profile is view only or not
```

Comment:
Is there a reason this is written twice?

Reply:
that's my bad I was going over all the components that had viewOnly and was copy pasting it in and ig I did it twice lol